exporter/clickhouseexporter/config.go

```diff
@@ -145,6 +172,48 @@ func (cfg *Config) shouldCreateSchema() bool {
 	return cfg.CreateSchema
 }
 
+func (cfg *Config) ValidateMetricTableNames() error {
+	if len(cfg.MetricsTableName) == 0 {
+		return nil
+	}
+	if cfg.areTableNamesSet() {
+		return fmt.Errorf("Warning: 'metrics_table_name' is deprecated, use 'metrics_tables' instead")
+	}
+	return nil
+}
+
+func (cfg *Config) buildTableNames() {
```

Comment:
I like that the metrics table naming flow is encapsulated within this function, see note about `areTableNamesSet` to see if we can remove the dependency on that function.

As noted in another comment, let's also add "Metric(s)" somewhere in this function name so it's clear *which* table names we're building